helix-term/src/application.rs

```diff
@@ -227,7 +227,11 @@ impl Application {
                         doc.set_selection(view_id, pos);
                     }
                 }
-                editor.set_status(format!("Loaded {} files.", nr_of_files));
+                editor.set_status(format!(
+                    "Loaded {} file{}.",
+                    nr_of_files,
+                    "s".repeat((nr_of_files != 1).into()) // avoid "Loaded 1 files." grammo
```

Comment:
I think this is clever but a little implicit. A regular `if` expression is more readable IMO: https://github.com/helix-editor/helix/blob/b12c65678aacc577b070c70307ef6fce528e4d85/helix-term/src/ui/statusline.rs#L302